login/login.js

```diff
@@ -0,0 +1,129 @@
+let mysql = require('mysql');
+
+let connectionConfig = require('./ConfigConnection.json');
+
+const connection = mysql.createConnection(connectionConfig);
+const LoginRequest  = require("./LoginRequest.js");
+const FrontQuery = require("./FrontQuery.js");
+const UserData = require("./UserData.js");
+
+
+const Sequelize = require("sequelize");
+const sequelize = new Sequelize("web","root","Edding16", {
+    dialect: "mysql",
+    host: "localhost",
+    logging: false,
+    define: {
+        timestamps: false,
+    }
+});
```

Comment:
I see a lot of space for abstracting things out in this file that don't really belong here, like db connections(look at the Repository pattern) and definitions for sequalization.

Reply:
yes,u are right,i have busy schedule and made this mistake,i will fix it as soon as posible